packages/terra-form-select/src/terra-dev-site/doc/form-select/SelectField.2.doc.jsx

```diff
@@ -26,6 +28,11 @@ const DocPage = () => (
       title: 'Select Field Example',
       example: <SelectFieldExample />,
       source: SelectFieldExampleSrc,
+    }, {
+      title: 'Maximum Selection',
```

Comment:
```suggestion
      title: 'Maximum Selection (applies to only \'multiple\' and \'tag\')',
```

 title: 'Clear', -> ' Ability to Clear Selection (applies to only 'default', 'combobox', and 'search')',

Reply:
Updated the example title: https://github.com/cerner/terra-core/pull/2244/commits/3c8375b9ae7e72f9eb55a40a15eab66819461ff7